.github/workflows/ci.yml

```diff
@@ -2,18 +2,14 @@ name: CI
 
 on:
   pull_request:
-    branches:
-      - master
   push:
-    branches:
-      - master
 
 jobs:
   build:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        java: [ '8', '11', '15' ]
+        java: [ '11' ]
```

Comment:
- dropped Java 8 support since https://github.com/brettwooldridge/HikariCP/commit/1991355e6b20be04be2b2d9ff816c32c38937fd2
- does not work with JDK 15 (or 16) 😢  

```
Error:  Exception in thread "main" java.lang.reflect.InvocationTargetException
Error:  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
Error:  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
Error:  	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
Error:  	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
Error:  	at java.instrument/sun.instrument.InstrumentationImpl.loadClassAndStartAgent(InstrumentationImpl.java:513)
Error:  	at java.instrument/sun.instrument.InstrumentationImpl.loadClassAndCallPremain(InstrumentationImpl.java:525)
Error:  Caused by: java.lang.RuntimeException: Class java/lang/UnknownError could not be instrumented.
Error:  	at org.jacoco.agent.rt.internal_28bab1d.core.runtime.ModifiedSystemClassRuntime.createFor(ModifiedSystemClassRuntime.java:140)
Error:  	at org.jacoco.agent.rt.internal_28bab1d.core.runtime.ModifiedSystemClassRuntime.createFor(ModifiedSystemClassRuntime.java:101)
Error:  	at org.jacoco.agent.rt.internal_28bab1d.PreMain.createRuntime(PreMain.java:55)
Error:  	at org.jacoco.agent.rt.internal_28bab1d.PreMain.premain(PreMain.java:47)
Error:  	... 6 more
Error:  Caused by: java.lang.NoSuchFieldException: $jacocoAccess
Error:  	at java.base/java.lang.Class.getField(Class.java:2092)
Error:  	at org.jacoco.agent.rt.internal_28bab1d.core.runtime.ModifiedSystemClassRuntime.createFor(ModifiedSystemClassRuntime.java:138)
Error:  	... 9 more
Error:  *** java.lang.instrument ASSERTION FAILED ***: "result" with message agent load/premain call failed at ./src/java.instrument/share/native/libinstrument/JPLISAgent.c line: 422
Error:  Aborted (core dumped)
```